doc/source/installation.rst

```diff
@@ -2,5 +2,211 @@
 Installation
 ############
 
-.. todo::
-    Migrate installation guide from Markdown wiki
+Lethe can be installed on Linux, Mac OS and under the Windows Subsystem for Linux (WSL). Lethe has been tested on numerous distributions, including Ubuntu 18.04 LTS, Ubuntu 20.04 LTS, Centos 7 and Manjaro. Lethe requires a modern version of the deal.II library. At the time of this writing, ''deal.II 9.3'' and ''deal.II 10.0pre'' (the master branch version) are supported. The compatibility with these two branches is ensured by Continuous Integration (CI) using Github Actions. A `dealii fork <https://github.com/lethe-cfd/dealii>`_ is maintained by the Lethe organization. This fork does not include any modification to the library, but is the latest deal.II version against which Lethe was tested. We work hard to ensure compatibility with the latest deal.II version and we do not modify the library except through pull request on the official deal.II repository.
+
+The installation of Lethe consists in three major steps:
+1. Installation of dealii dependency (mpi, numdiff, p4est, trilinos and METIS)
+2. Installation of the dealii library
+3. Installation of lethe
+
+There are two methods to install the dependencies. They can be either installed manually or through ``candi``. The candi.sh shell script can download, configure, build, and install all the dependencies along with dealii. **This is the easiest way**, since it requires much less manual intervention. Even if you do not want to use candi to install deal.II, you can also use it the dependencies. This is by far the easiest way to proceed. On a single core computer with 8Gb of ram, count up to 8 hours for a first installation, and 3 hours for a dealii update. On a machine with 16 cores and 32 Gb or ram, this process will take less than an hour or so. The installation of deal.II and its dependencies (especially Trilinos), can be extremely ram consuming. Installation on a machine with less than 8Gb of ram is difficult at best, impossible at worst.
```

Comment:
```suggestion
There are two methods to install the dependencies. They can be either installed manually or through ``candi``. The candi.sh shell script can download, configure, build, and install all the dependencies along with deal.II. **This is the easiest way**, since it requires much less manual intervention. Even if you do not want to use candi to install deal.II, you can use it for the dependencies. This is by far the easiest way to proceed. On a single core computer with 8GB of RAM, count up to 8 hours for a first installation, and 3 hours for a deal.II update. On a machine with 16 cores and 32GB of RAM, this process will take less than an hour or so. The installation of deal.II and its dependencies (especially Trilinos), can be extremely RAM consuming. Installation on a machine with less than 8GB of RAM is difficult at best, impossible at worst.
```